monitor/monitoring-dam-generator-service/README.md

```diff
@@ -0,0 +1,30 @@
+# SeaClouds - Monitoring DAM Generator Service
+
+The main goal of this service within the SeaClouds Platform is to provide an automatic mechanism to generate all the required monitoring artifacts for an application managed by SeaClouds. The Monitoring DAM Generator Service basically wraps the Monitoring DAM Generator Core.
+Given an Abstract Deployment Plan (ADP) specified in the SeaClouds TOSCA syntax, the Monitoring DAM Generator Core builds up a set of monitoring rules to be then installed into Tower 4Clouds platform in order to retrieve and filter at runtime the monitring data sent by the deployed data collectors. The Monitoring DAM Generator Core also builds a set of bash scripts suitable to deploy all the necessary data collectors to collect the metrics requested by the generated monitoring rules. Finally it generates all the required pieces of yaml to be added into the SeaClouds DAM in order to be able to deploy the data collectors. This are generated according to the current DAM definition ( https://docs.google.com/document/d/1Z9RdaWn54IomgI-jCiw-tdutsCdDoHfk67PNUASOFmY/edit?disco=AAAAAiYOzxM ) and each TOSCA script is associated to one of the bash script previously generated.
+
+The Monitoring DAM Generator Service is a Dropwizard application and by default it listen on port 8177.
+
+# Usage
+
+The Monitoring DAM Generator basically exhibits three distinct endpoints:
+
+POST /damgen : this is the main endpoint which generates the application specific monitoring information. The request body must be the yaml representing the application ADP. It returns a JSON object with a single field containing an ID which can be used later on in order to retrieve from the Monitoring DAM Generator Service itself the generated rules and data collectors deployment scripts.
+
+GET /{applicationId}/generatedDam : this interface returns a JSON object in which each field represent a TOSCA based deployment script for one of the data collectors which have to be deployed with the application.
+
+GET /{applicationId}/monitoringRule : this interface return a JSON object with a single field contaning all the monitoring rules that have to be installed in order to monitor the application.
+
+In order to run the service one need to use the following lines of code:
+
+    export MONITOR_IP=<MONITORING-MANAGER-IP>    
+    export MONITOR_PORT=<MONITORING-MANAGER-PORT>    
```

Comment:
The config.yaml is actually retrieving this values from the previously exported env vars. It looks like the following:

server:
  type: simple
  applicationContextPath: /
  connector:
    type: http
    port: 8177
monitorIp: ${MONITOR_IP}
monitorPort: ${MONITOR_PORT}

Is this the correct approach?